.rspec

```diff
@@ -1,3 +1,5 @@
+--require manageiq/spec/spec_helper
```

Comment:
Do we really want this? Seems like a coupling to the main project that doesn't need to happen.

Reply:
My goal was to get this closer to how the rest of the plugins work.

Reply:
Ah, ok; My thought still stands but if the rest of them do this already then ¯\\\(°_o)/¯ :shipit: